bundles/org.openhab.ui.basic/web-src/smarthome.js

```diff
@@ -2372,10 +2374,8 @@
 		}
 
 		function cleanValue(value) {
-			var prefix = _t.prefixField !== null ? _t.prefixField.innerHTML : "";
-			var postfix = _t.postfixField !== null ? _t.postfixField.innerHTML : "";
-			var newValue = value.startsWith(prefix) ? value.substr(prefix.length) : value;
-			newValue = value.endsWith(postfix) ? newValue.substr(0, newValue.lastIndexOf(postfix)) : newValue;
+			var newValue = (_t.prefix && value.startsWith(_t.prefix)) ? value.substr(_t.prefix.length) : value;
+			newValue = (_t.postfix && value.endsWith(_t.postfix)) ? newValue.substr(0, newValue.lastIndexOf(_t.postfix)) : newValue;
```

Comment:
Is condition `_t.prefix` equivalent to `_t.prefix !== null` ? I am just not 100% sure.

Reply:
null, undefined or empty will all produce false, which is OK. This avoids issues with the following calls on it.